### PR TITLE
Set up & run linter based on previous configs

### PR DESCRIPTION
**Pull request by MickeyDang**
# Description

- Added commands in package.json
- Ran fix

# Screenshots

Please include screenshots if you worked on a front end feature in desktop view + mobile view (if you worked on it) 

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,13 @@
...
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "yarn lint:prettier && yarn lint:eslint",
+    "lint:prettier": "prettier --check src/",
+    "lint:eslint": "eslint src/ --cache --format stylish",
+    "fix": "yarn fix:eslint && yarn fix:prettier",
+    "fix:prettier": "prettier --write src/",
+    "fix:eslint": "eslint src/ --format stylish --fix"
```
Comment (AbhiPrasad) [inline package.json:41]: @MickeyDang  hmmm this look familiar 🤔 

image

Comment (AbhiPrasad) [reply package.json:41]: dw though i stole it from Sentry

https://github.com/getsentry/sentry-javascript/blob/master/packages/browser/package.json#L68-L73

**Comment by MickeyDang** [comment]
Why write my own code when there are numerous `please run yarn fix` memes to extend.

File: package.json (hunk 1 above)
Comment (MickeyDang) [reply package.json:41]: Why write my own code when there are numerous `please run yarn fix` memes to extend.

**Pull request merged**